.github/workflows/android-master.yml

```diff
@@ -77,8 +77,9 @@ jobs:
 
       # Stores the generated AAR to an outputs folder
       - name: Store AAR
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
```

Comment:
tbh i'm not even sure what this does 🤦

Reply:
Hm nevermind, the artifact IS meant to be there. But I dont see it in our builds, now I'm wondering why https://github.com/actions/upload-artifact?tab=readme-ov-file#where-does-the-upload-go

Reply:
Ah I see some github chatter that this was a known issue with `upload-artifact` v3, meaning if we keep this step here but update it to v4 currently, the artifact should always append to the action's run in the UI so we can download for potential debugging on failed master runs.
Minor but I think worth keeping in